code/game/turfs/change_turf.dm

```diff
@@ -38,6 +38,12 @@ GLOBAL_LIST_INIT(blacklisted_automated_baseturfs, typecacheof(list(
 		T.setDir(dir)
 	return T
 
+/turf/open/copyTurf(turf/T, copy_air = FALSE)
```

Comment:
This is already defined elsewhere, probably should move it into this file though

Reply:
fixed